templates/taxi/car_detail.html

```diff
@@ -12,6 +12,16 @@ <h1>
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+  {% if user not in car.drivers.all %}
+    <a href="{% url 'taxi:driver-add' car.id %}" class="btn btn-primary">
+        Assign me to this car
+    </a>
+  {% else %}
+    <a href="{% url 'taxi:driver-remove' car.id %}" class="btn btn-danger">
+        Delete me from this car
+    </a>
+  {% endif %}
```

Comment:
The condition `{% if user not in car.drivers.all %}` checks if the current user is not a driver of the car. Ensure that `user` is correctly passed to the template context. If `user` is not available, this condition will not work as expected. Consider verifying that the `user` context variable is correctly set in the view rendering this template.